apps/web/public/static/locales/en/common.json

```diff
@@ -2769,5 +2769,7 @@
   "salesforce_route_to_custom_lookup_field": "Route to a user that matches a lookup field on an account",
   "salesforce_option": "Salesforce Option",
   "lookup_field_name": "Lookup Field Name",
+  "exclude_emails_that_contain" : "Exclude emails that contain ...",
+  "exclude_emails_match_found_error_message" : "Please enter a valid work email address",
```

Comment:
this feature is primarily to throw this error: 
<img width="279" alt="image" src="https://github.com/user-attachments/assets/b6d018f9-04a8-496d-88d7-593404720942">